wp-content/themes/core/pcss/components/_cta.pcss

```diff
@@ -5,31 +5,69 @@
  * ----------------------------------------------------------------------------- */
 
 .c-cta,
-a.c-cta,
-.t-content .c-cta {
-	border-bottom: none;
-	color: var(--color-link);
+a.c-cta {
```

Comment:
Fwiw, I prefer to setup all button styles (including the "this is a text link" anchor variation, we should also chat with design about the two variations of anchor) as a `c-btn-` and not prescribe a specific action style to the cta on it's own as the cta component itself will very likely need to change on what action variation it uses based on context of implementation.

Reply:
Added as `c-btn-text` and switched the CTA back to its previous styles